iceoryx_posh/include/iceoryx_posh/internal/popo/ports/subscriber_port_user.hpp

```diff
@@ -19,6 +19,7 @@
 #include "iceoryx_posh/internal/popo/ports/base_port.hpp"
 #include "iceoryx_posh/internal/popo/ports/subscriber_port_data.hpp"
 #include "iceoryx_posh/mepoo/chunk_header.hpp"
+#include "iceoryx_posh/popo/subscriber_options.hpp"
```

Comment:
Remove this include?

Reply:
This makes it kind of easier. It would be needed in `subriber_port_data.hpp` but the `subscriber_options.hpp` need a default value from `subriber_port_data.hpp` and then there would be a cyclic dependency. Currently there is a forward declaration to `SubscriberOptions`. One idea would be to either include `subscriber_options.hpp` at the end of `subriber_port_data.hpp` or move the `ChunkQueueData_t` alias to `iceoryx_posh_types.hpp`